docs/lib/datastore/fragments/android/getting-started/70_querySnippet.md

```diff
@@ -0,0 +1,22 @@
+<amplify-block-switcher>
+<amplify-block name="Java">
+
+```java
+Amplify.DataStore.query(Post.class,
+    result -> Log.i("MyAmplifyApp", "Posts retrieved successfully"),
+    error -> Log.e("MyAmplifyApp",  "Error retrieving posts", error)
+);
```

Comment:
A `save(...)` or `delete(...)` is useful, without considering the result value.

However, a `query(...)` is _useless, unless you do something with the result value. So, this example should be updated to grok the `result` data.

Instead of iterating over all of the results, which will start to make the example long, we could just show that it is an iterator, like this:

```suggestion
Amplify.DataStore.query(Post.class,
    queryMatches -> {
        if (queryMatches.hasNext()) {
            Log.i("MyAmplifyApp", "Successful query, found posts.");
        } else {
            Log.i("MyAmplifyApp", "Successful query, but no posts.");
        }
    },
    error -> Log.e("MyAmplifyApp",  "Error retrieving posts", error)
);
```